helm/reana-dev/values.yaml

```diff
@@ -0,0 +1,18 @@
+# REANA components pointing to `latest`, locally built master branch
```

Comment:
`reana-dev` subdirectory only holds `values.yaml`.  Couldn't we simply have `values-dev.yaml`?  Thinking of future extensions, we won't create a new directory for each configuration... It would scale better if we could have something like:

```
helm/reana/charts/...
helm/configurations/minikube-production.yaml
helm/configurations/minikube-development.yaml
helm/configurations/minikube-develelopment-vc3.yaml
...
```

(... also, we have `dev-build` in Makefile, `BUILD_TYPE` uses "development"... I used longer version "development" here, but we could also standardise into "dev"...)